lib/core/certificates/certs/readme.md

```diff
@@ -3,3 +3,7 @@ These certificates are copied from https://github.com/ConsultingMD/paloaltonetwo
 They are used by the GlobalProtect VPN DLP MITM inspector.
 
 If these certs expire John Walsh is probably the person to talk to.
+
+The mozilla.pem file contains all the certs from
```

Comment:
from where?

Reply:
Derp, I went to go get the link to the document about where these certs were from, got distracted, and just left it blank. I'll fill that in.